utils/wif.go

```diff
@@ -63,7 +65,7 @@ func (w *WIF) IsForNet(net *params.Params) bool {
 	return w.netID == net.PrivateKeyID
 }
 
-// DecodeWIF creates a new WIF structure by decoding the string encoding of
+// DecodePassWIF creates a new WIF structure by decoding the string encoding of
 // the import format.
 //
 // The WIF string must be a base58-encoded string of the following byte
```

Comment:
1. Please re-arrange the comments, the original comments should owned by `DecodeWIF` not `DecodePassWIF`
2. Please find a better name for the wif format instead `format=pastwif` . ex. `kahf_internal` `kahf_legacy` `kafh_v0.9`
3. Decode`Pass`WIF ? `Past`String ? Please keep using the unique name. the current naming is confusing.